src/Entities/User.php

```diff
@@ -0,0 +1,16 @@
+<?php
+readonly class User
```

Comment:
I'll ignore this User stuff for now....

In future, please avoid putting any work from future stories in PRs!